docs/reference-manual/native-image/assets/build-output-schema-v0.9.1.json

```diff
@@ -0,0 +1,607 @@
+            "objects": {
+              "type": "object",
+              "default": {},
+              "title": "Object statistics",
+              "required": [
+                "count"
+              ],
+              "properties": {
+                "count": {
+                  "type": "integer",
+                  "default": 0,
+                  "title": "Number of objects in the image heap"
+                }
+              }
+            },
+            "resources": {
+              "type": "object",
+              "default": {},
+              "title": "Resource statistics",
+              "required": [
+                "count",
+                "bytes"
+              ],
+              "properties": {
+                "count": {
+                  "type": "integer",
+                  "default": 0,
+                  "title": "Number of resources embedded in the image"
+                },
+                "bytes": {
+                  "type": "integer",
+                  "default": 0,
+                  "title": "The number of bytes used for resource data"
+                }
+              }
+            }
+          }
+        },
+        "debug_info": {
+          "type": "object",
+          "default": {},
+          "title": "Debug info statistics",
+          "required": [
+            "bytes"
+          ],
+          "properties": {
+            "bytes": {
+              "type": "integer",
+              "default": 0,
+              "title": "The number of bytes used for debug info"
+            }
+          }
+        },
+        "runtime_compiled_methods": {
+          "type": "object",
+          "default": {},
+          "title": "Statistics on runtime compiled methods (optional)",
+          "required": [
+            "count",
+            "graph_encoding_bytes"
+          ],
+          "properties": {
+            "count": {
+              "type": "integer",
+              "default": 0,
+              "title": "Number of runtime compiled methods"
+            },
+            "graph_encoding_bytes": {
+              "type": "integer",
+              "default": 0,
+              "title": "The number of bytes used for graph encodings bytes"
+            }
+          }
+        }
+      },
+      "examples": [
+        {
+          "total_bytes": 13057934,
+          "code_area": {
+            "bytes": 4610048,
+            "compilation_units": 67007
+          },
+          "image_heap": {
+            "bytes": 7307264,
+            "objects": {
+              "count": 83319
+            },
+            "resources": {
+              "count": 134,
+              "bytes": 10200
+            }
+          },
+          "debug_info": {
+            "bytes": 1140622
+          }
+        }
+      ]
+    },
+    "resource_usage": {
+      "type": "object",
+      "default": {},
+      "title": "Resource usage statistics",
+      "required": [
+        "cpu",
+        "garbage_collection",
+        "memory",
+        "time"
```

Comment:
This should be `total_secs` instead of `time`. Causes the following validation error in gates:

```
jsonschema.exceptions.ValidationError: 'time' is a required property
```